src/ui/src/js/controllers/admin_garden.js

```diff
@@ -53,15 +53,32 @@ export default function adminGardenController(
     loadGardens();
   };
 
-  EventService.addCallback('admin_system', (event) => {
+  EventService.addCallback('admin_garden', (event) => {
     switch (event.name) {
       case 'GARDEN_CREATED':
-        break;
+        $scope.data.push(event.payload);
+        $apply();
       case 'GARDEN_REMOVED':
-        break;
```

Comment:
Why was this removed? `switch` statements needs `break`

Reply:
let's just say it has been a while since someone has done switch statements...